config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -502,7 +504,15 @@ periodics:
       - --kubemark-nodes=100
       - --provider=gce
       - --test=false
-      - --test_args=--ginkgo.focus=\[Feature:Performance\] --gather-resource-usage=true --gather-metrics-at-teardown=true
+      - --test_args=--ginkgo.focus=xxxx
```

Comment:
Is that really needed?

Reply:
The problem is that there is a hardcoded ginkgo test run when kubemark flag is provided. I didn't want to change this logic of kubetest to not affect other kubemerk jobs. This line is just make test run for empty tests set.